### ISSUE TITLE
Set icon for linux

### ISSUE THREAD

**Issue by stevenvachon** [nwutils/nw-builder#68]
... for specifying an app icon. GNOME and KDE both support PNG and SVG files for icons.

http://standards.freedesktop.org/desktop-entry-spec/latest/index.html

**Comment by Bloodyaugust** [comment]
+1 for this. Sucks that we can specify it for 2/3 platforms.

**Comment by hotrush** [comment]
+1

**Comment by stkevintan** [comment]
+1

**Comment by adam-lynch** [comment]
There aren't many maintainers on this project. I personally haven't a clue about this topic. Pull-requests are more than welcome :smile:

**Comment by maxrimue** [comment]
@adam-lynch I would like to create a PR but unfortunately don't have time right now to get to know your code, as I'm also not as good in JavaScript yet. However, all you need to do is, to create a `.desktop` file for Linux builds (I think this is comparable to the plist for OS X?) which needs to be placed in the root of the resulting build folder. With the `.desktop` file, the resulting applications will also be treated by all Linux distros as applications, perhaps more than they do right now.

The .desktop file has a structure like this:
```
[Desktop Entry]
Type=Application                          # Indicates the type
Version=1.0                               # The version of the desktop entry specification to which this file complies
Name=nw.jsApp                             # The name of the application
Comment=Some nw.js app I created          # A comment which can/will be used as a tooltip
Exec=myApp                                # The executable of the application.
Icon=someIcon                             # The name of the icon that will be used to display this entry
Terminal=false                            # Describes whether this application needs to be run in a terminal or not
Categories=Education;Languages;nwjs;      # Describes the categories in which this entry should be shown
```

You could introduce a new universal `icon` property for the API with that, I would prefer `linuxIcon` though as most Distros seem to support `png` and `svg` for their logo.
Thanks for creating this module, helps a lot. I hope you consider putting in this functionality; ask me when you need something.

**Comment by vleong2332** [comment]
+1

**Comment by ayushmxn** [comment]
Resolved by #661
